solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/FieldValueFeature.java

```diff
@@ -113,9 +113,10 @@ public float score() throws IOException {
             }
           }
         } catch (final IOException e) {
-          // TODO discuss about about feature failures:
-          // do we want to return a default value?
-          // do we want to fail?
+          throw new FeatureException(
```

Comment:
As mentioned in person, this just changes the feature throwing exception to be consistent with SolrFeature.